raco/backends/sql/catalog.py

```diff
@@ -110,6 +111,8 @@ def _convert_unary_expr(self, cols, expr, input_scheme):
             return func.max(input)
         if isinstance(expr, expression.MIN):
             return func.min(input)
+        if isinstance(expr, expression.SUM):
```

Comment:
This should be covered by https://github.com/uwescience/raco/issues/502 and removed from this PR.